xclim/indices/_hydrology.py

```diff
@@ -279,3 +283,120 @@ def melt_and_precip_max(
+        It accepts the same arguments as :py:func:`xclim.indices.generic.select_time`.
+
+    Returns
+    -------
+    xr.DataArray
+        Normalized Qp, which is the p th percentile of daily streamflow normalized by the median flow.
+
+    References
+    ----------
+    1. Addor, Nans & Nearing, Grey & Prieto, Cristina & Newman, A. & Le Vine, Nataliya & Clark, Martyn. (2018). A Ranking of Hydrological Signatures Based on Their Predictability in Space. Water Resources Research. 10.1029/2018WR022606.
+    2. Clausen, B., & Biggs, B. J. F. (2000). Flow variables for ecological studies in temperate streams: Groupings based on covariance. Journal of Hydrology, 237(3–4), 184–197. https://doi.org/10.1016/S0022-1694(00)00306-1
+
+    """
+    qt = select_time(q, **indexer)
+    qp = qt.quantile(p, dim="time")
+    q_median = qt.median(dim="time")
+    out = qp / q_median
+    out.attrs["units"] = "1"
+    return out
+
+
+@declare_units(q="[discharge]")
+def high_flow_frequency(
+    q: xr.DataArray, threshold_factor: int = 9, freq: str = "A-SEP", **indexer
+) -> xr.DataArray:
+    """
+    High flow frequency.
+
+    Calculate the number of days in a given period with flows greater than a specified threshold, given as a
+    multiple of the median flow. By default, the period is the water year starting on 1st October and ending on
+    30th September, as commonly defined in North America.
+
+    Parameters
+    ----------
+    q : xr.DataArray
+        Daily streamflow data.
+    threshold_factor : int
+        Factor by which the median flow is multiplied to set the high flow threshold, default is 9.
+    freq : str, optional
+        Resampling frequency, default is 'A-SEP' for water year ending in September.
+    indexer
+        Indexing parameters to perform a temporal subset of the data.
+        It accepts the same arguments as :py:func:`xclim.indices.generic.select_time`.
+
+    Returns
+    -------
+    xr.DataArray
+        Calculated mean of high flow days per water year
+
+    References
+    ----------
+    1. Addor, Nans & Nearing, Grey & Prieto, Cristina & Newman, A. & Le Vine, Nataliya & Clark, Martyn. (2018). A Ranking of Hydrological Signatures Based on Their Predictability in Space. Water Resources Research. 10.1029/2018WR022606.
+    2. Clausen, B., & Biggs, B. J. F. (2000). Flow variables for ecological studies in temperate streams: Groupings based on covariance. Journal of Hydrology, 237(3–4), 184–197. https://doi.org/10.1016/S0022-1694(00)00306-1
+    """
+    median_flow = q.median(dim="time")
+    with xr.set_options(keep_attrs=True):
+        threshold = threshold_factor * median_flow
+
+    sel = select_time(q, **indexer)
+    out = threshold_count(sel, ">", threshold, freq=freq)
+    return to_agg_units(out, q, "count")
+
+
+@declare_units(q="[discharge]")
+def low_flow_frequency(
+    q: xr.DataArray, threshold_factor: float = 0.2, freq: str = "A-SEP", **indexer
```

Comment:
```suggestion
    q: xr.DataArray, threshold_factor: float = 0.2, freq: str = "YS-SEP", **indexer
```
Same comment as above.